python-frontend/src/test/java/org/sonar/python/tree/IPythonTreeMakerTest.java

```diff
@@ -39,21 +40,23 @@ public void line_magic() {
     var parse = parseIPython("print(b)\n" +
       "a = %alias showPath pwd && ls -a\n", treeMaker::fileInput);
     var statementList = findFirstChildWithKind(parse, Tree.Kind.STATEMENT_LIST);
```

Comment:
By the way, instead of this, you can access the statement list simply with `parse.statements().statements()` which I think is a bit more explicit.

Reply:
great! done)